frontend/src/api/opsAPI.js

```diff
@@ -212,6 +212,24 @@ export const opsApi = createApi({
             }),
             invalidatesTags: ["Cans"]
         }),
+        addCanFundingBudgets: builder.mutation({
+            query: ({ data }) => ({
+                url: `/can-funding-budgets/`,
+                method: "POST",
+                headers: { "Content-Type": "application/json" },
+                body: data
+            }),
+            invalidatesTags: ["Cans"]
+        }),
+        updateCanFundingBudget: builder.mutation({
+            query: ({ id, data }) => ({
+                url: `/can-funding-budgets/${id}`,
+                method: "PATCH",
+                headers: { "Content-Type": "application/json" },
+                body: data
+            }),
+            invalidatesTags: ["Cans"]
```

Comment:
I think you will want to add `CanFunding` tag to this list.  That way when you update the funding the `getCanFundingSummary` will re-fetch.

Reply:
Good catch!